packages/flutter/test/rendering/table_test.dart

```diff
@@ -54,7 +54,7 @@ void main() {
     table.setFlatChildren(6, children);
     layout(table, constraints: const BoxConstraints.tightFor(width: 100.0));
 
-    const double expectedWidth = 100.0 / 6;
+    final double expectedWidth = (100.0 / 6).ceilToDouble();
```

Comment:
Why should the expected width be rounded?

Reply:
Because my fix is to ceil the decimal to the integer, the cause of issue is it has gaps between columns if cell width is not decimal.
![image](https://user-images.githubusercontent.com/5061845/76271181-a464ed80-62b2-11ea-9aa1-c45c01402998.png)

Reply:
I don't think just rounding this to the next biggest integer is the right thing to do here. Wouldn't this cause some overlap between the cells now?

Reply:
@goderbauer there may have overlap but not exceed 1 point, and I don't know how Skia handle decimal width, how to render if width is smaller than 1 point, maybe it also do some rounding?

Reply:
But even a 1px overlap will look bad if things are not fully opaque. Like I said, I don't think this is the right fix.

Reply:
@goderbauer We can also adjust the position of the cells. And apparently rendering also do some rounding, because we see the gap :)